src/apps/bridge/app_config.h

```diff
@@ -12,6 +12,7 @@ constexpr const char *SUBSAMPLE_ENABLED = "subsampleEnabled";
 constexpr const char *BRIDGE_POWER_CONTROLLER_ENABLED = "bridgePowerControllerEnabled";
 constexpr const char *ALIGNMENT_INTERVAL_5MIN = "alignmentInterval5Min";
 constexpr const char *TRANSMIT_AGGREGATIONS = "transmitAggregations";
+constexpr const char *SAMPLES_PER_REPORT = "samplesPerReport";
```

Comment:
This isn't used by this PR , but will be used by the "report builder" which will come next. (This PR originally implemented this but took a drastic turn. Since it will eventually be needed I decided it was okay to leave it)